docs/configuration-options.md

```diff
@@ -1,33 +1,74 @@
-| `exclude("…")`                      | Applies a regular expression to exclude benchmarks that match the substring in their fully qualified names.                          | Regex pattern                        | -                        |
-| `param("name", "value1", "value2")` | Assigns values to a public mutable property, annotated with `@Param`.                                                                | Any string values                    | @Param                   |
-| `reportFormat`                      | Defines the benchmark report's format options.                                                                                       | "json", "csv", "scsv", "text"        | -                        |
-
-## Expert Configuration Options: The Power Settings
-
-The power of kotlinx-benchmark extends beyond basic settings. Delve into platform-specific options for tighter control over your benchmarks:
-
-| Option                                        | Platform               | Description                                                                                                            | Possible Values                                  | Corresponding Annotation |
-| --------------------------------------------- | ---------------------- | ---------------------------------------------------------------------------------------------------------------------- | ------------------------------------------------ | ------------------------ |
-| `advanced("nativeFork", "value")`             | Kotlin/Native          | Executes iterations within the same process ("perBenchmark") or each iteration in a separate process ("perIteration"). | "perBenchmark", "perIteration"                   | -                        |
-| `advanced("nativeGCAfterIteration", "value")` | Kotlin/Native          | Triggers garbage collection after each iteration when set to `true`.                                                   | `true`, `false`                                  | -                        |
-| `advanced("jvmForks", "value")`               | Kotlin/JVM             | Determines how many times the harness should fork.                                                                     | "0" (no fork), "1", "definedByJmh" (JMH decides) | @Fork                    |
-| `advanced("jsUseBridge", "value")`            | Kotlin/JS, Kotlin/Wasm | Disables the generation of benchmark bridges to stop inlining optimizations when set to `false`.                       | `true`, `false`                                  | -                        |
+This is a comprehensive guide to configuration options that help fine-tune your benchmarking setup to suit your specific needs.
+The `configurations` section of the `benchmark` block serves as the control center for setting the parameters of your benchmark profiles.
```

Comment:
Makes sense to mention somewhere in this document that the library has the `"main"` configuration profile created already. And one could configure it as well.

Reply:
Thank you for pointing this out. I have updated the documentation to include this.